build.gradle

```diff
@@ -80,6 +88,8 @@ dependencies {
 
 	implementation "me.shedaniel.cloth:cloth-config-neoforge:${cloth_config_version}"
 
+	implementation "com.blakebr0.mysticalagriculture:MysticalAgriculture:${mystical_agriculture_version}"
```

Comment:
I'd rather the mod not always be included in the dev environment. Add a property in `gradle.properties` to enable/disable this like how I do with EMI, MI Tweaks, Accessories, and Curios.